test_framework/Jenkinsfile

```diff
@@ -43,23 +43,23 @@ node {
                 sh "docker stop ${JOB_NAME}${BUILD_NUMBER}"
                 sh "docker rm -v ${JOB_NAME}${BUILD_NUMBER}"
                 sh "docker rmi ${imageName}"
-
-                notifyBuild(currentBuild.result)
+                summary_msg = "\nTest Summary - Total: ${summary.totalCount}, Failures: ${summary.failCount}, Skipped: ${summary.skipCount}, Passed: ${summary.passCount}"
```

Comment:
Let's do `Failure, Skipped, Total`. No need for passing since it's the remaining count.

Do we know how long it took? Want to add that to the report as well if possible.

Reply:
Fixed